### ISSUE TITLE
add number_metabomatching_permutations as an option

### ISSUE THREAD

**Issue by mattiat** [BergmannLab/metabomodules-docker#3]
In metabomodules.py  “number_metabomatching_permutations” variable is hardcoded please add it to parsers’ argument as it is impossible to change it for users using the docker. Also add a warning if it is set to less than 9999 that for the filtering to be reliable this variable should be set to 9999.

**Comment by mattiat** [comment]
I don't think we should allow end users to change this. Why allowing them to change it and them warning them they should not?

**Comment by mattiat** [comment]
@BKhalili can you explain your motivation?

**Comment by BKhalili** [comment]
Right now it is 99 in the code. I think you should allow for the change if a user wants to have a quick check. Like in our case that we want to check the code and results.

**Comment by BKhalili** [comment]
There is an error when trying to run the docker and changing the number of permutations.
In line 176 of metabomodules.py when printing the warning  str(number_metabomatching_permutations) should replace number_metabomatching_permutations.

**Comment by mattiat** [comment]
Fixed, thanks!
